spyder/widgets/about.py

```diff
@@ -105,71 +157,132 @@ def __init__(self, parent):
             the <a href="http://www.famfamfam.com/lab/icons/silk/">FamFamFam
             Silk icon set</a> 1.3 (&copy; 2006 Mark James; CC-BY 2.5), and
             the <a href="https://www.kde.org/">KDE Oxygen icons</a>
-            (&copy; 2007 KDE Artists; LGPL 3.0+).</small>
+            (&copy; 2007 KDE Artists; LGPL 3.0+).
+            </p>
+            <p>
+            Splash screen photo by
+            <a href="https://unsplash.com/@benchaccounting?utm_source=
+            unsplash&utm_medium=referral&utm_content=creditCopyText">Bench
+            Accounting</a> on <a href="https://unsplash.com/?utm_source=
+            unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash
+            </a>
             </p>
             <p>
             See the <a href="{github_url}/blob/master/NOTICE.txt">NOTICE</a>
             file for full legal information.
             </p>
             </div>
             """).format(
-                spyder_ver=versions['spyder'],
-                revision=revlink,
-                website_url=__website_url__,
                 github_url=__project_url__,
-                trouble_url=__trouble_url__,
-                forum_url=__forum_url__,
-                python_ver=versions['python'],
-                bitness=versions['bitness'],
-                qt_ver=versions['qt'],
-                qt_api=versions['qt_api'],
-                qt_api_ver=versions['qt_api_ver'],
-                os_name=versions['system'],
-                os_ver=versions['release'],
                 font_family=font_family,
                 font_size=font_size,
             )
         )
-        self.label.setWordWrap(True)
-        self.label.setAlignment(Qt.AlignTop)
-        self.label.setOpenExternalLinks(True)
-        self.label.setTextInteractionFlags(Qt.TextBrowserInteraction)
-        self.label.setFixedWidth(350)
+
+        for label in [self.label_overview, self.label_community,
+                      self.label_legal]:
+            label.setWordWrap(True)
+            label.setAlignment(Qt.AlignTop)
+            label.setOpenExternalLinks(True)
+            label.setTextInteractionFlags(Qt.TextBrowserInteraction)
+            label.setContentsMargins(15, 0, 25, 0)
 
         icon_filename = "spyder_about"
         pixmap = QPixmap(get_image_path(icon_filename))
         self.label_pic = QLabel(self)
         self.label_pic.setPixmap(
-            pixmap.scaledToWidth(64, Qt.SmoothTransformation))
-        self.label_pic.setAlignment(Qt.AlignTop)
+            pixmap.scaledToWidth(100, Qt.SmoothTransformation))
+        self.label_pic.setAlignment(Qt.AlignBottom)
+        self.info = QLabel((
+            """
+            <div style='font-family: "{font_family}";
+                font-size: {font_size}pt;
```

Comment:
```suggestion
                font-size: {font_size};
```

`pt` is already included in `DialogStyle.ContentFontSize`